src/cli/feature_parser.js

```diff
@@ -11,6 +11,7 @@ export default class Parser {
       gherkinDocument = gherkinParser.parse(source)
     } catch (error) {
       error.message += '\npath: ' + uri
+      error.stack += '\npath: ' + uri
```

Comment:
This is adding the path to the very end of the stack. Can we add a test for this as to what the expected error stack should look like?